packages/framework-core/src/services/graphql/graphql-query-generator.ts

```diff
@@ -78,6 +79,26 @@ export class GraphQLQueryGenerator {
     return queries
   }
 
+  private generateFilterListedQueries(): GraphQLFieldConfigMap<unknown, GraphQLResolverContext> {
+    const queries: GraphQLFieldConfigMap<unknown, GraphQLResolverContext> = {}
+    for (const name in this.targetTypes) {
+      const type = this.targetTypes[name]
+      const graphQLType = this.typeInformer.getGraphQLTypeFor(type.class)
+      queries[`List${inflected.pluralize(name)}`] = {
+        type: new GraphQLObjectType({
+          name: `${type.class.name}Connection`,
```

Comment:
Just wondering because I'm having a hard time understanding what the `Connection` name means here. What does it refer to? Is this a connection to something? 

Wouldn't using "Result" convey a closer meaning to what it is?

Reply:
Also, I think you can use just `${name}...` instead of `${type.class.name}`

Reply:
Hey @alvaroloes, the idea of using `Connection` is to match the standard GraphQL type used for the result of a paginated query https://graphql.org/learn/pagination/#complete-connection-model

Reply:
I still don't understand why GraphQL called it Connection. But it has some "upvotes" because it is the standard in GraphQL, I  think it is not a bad idea.